model_analyzer/record/metrics_mapper.py

```diff
@@ -0,0 +1,57 @@
+# Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from .gpu_free_memory import GPUFreeMemory
+from .gpu_used_memory import GPUUsedMemory
+from .gpu_utilization import GPUUtilization
+from .cpu_used_memory import CPUUsedMemory
+from .cpu_free_memory import CPUFreeMemory
+from .perf_throughput import PerfThroughput
+from .perf_latency import PerfLatency
+
+
+class MetricsMapper:
+    """
+    Class maps metrics to record types
+    """
+
+    tag_to_metric = {
+        "Throughput": PerfThroughput,
```

Comment:
I think there is no need to make the key human readable. Can we make all the keys lower case and use underscore instead of spaces?

Reply:
The idea, was that eventually you'd be able to select metrics from the config file in which case it should be human readable no?

Reply:
Yes, I think we can do that even with the small cases. But it is not that important.